tutorials-v5/lectures/Lecture-5-Parametric-Amplifier.md

```diff
@@ -416,12 +416,12 @@ def plot_covariance_matrix(V, ax):
 
     ax.view_init(azim=-40, elev=60)
     ax.bar3d(xpos, ypos, zpos, dx, dy, dz, color=colors)
-    ax.axes.w_xaxis.set_major_locator(plt.IndexLocator(1, -0.5))
-    ax.axes.w_yaxis.set_major_locator(plt.IndexLocator(1, -0.5))
-    ax.axes.w_xaxis.set_ticklabels(("$q_-$", "$p_-$", "$q_+$", "$p_+$"),
-                                   fontsize=20)
-    ax.axes.w_yaxis.set_ticklabels(("$q_-$", "$p_-$", "$q_+$", "$p_+$"),
-                                   fontsize=20)
+    ax.axes.xaxis.set_major_locator(plt.IndexLocator(1, -0.5))
+    ax.axes.yaxis.set_major_locator(plt.IndexLocator(1, -0.5))
+    ax.axes.xaxis.set_ticklabels(("$q_-$", "$p_-$", "$q_+$", "$p_+$"),
+                                 fontsize=12)
+    ax.axes.yaxis.set_ticklabels(("$q_-$", "$p_-$", "$q_+$", "$p_+$"),
+                                 fontsize=12)
```

Comment:
This (as well as a few other things) still produces few warnings for me, but I guess it is fine for now. When Max or someone goes through all the notebooks, let's keep in mind that we should try to get rid of warnings?